cartographer/mapping/sparse_pose_graph.h

```diff
@@ -114,7 +114,7 @@ class SparsePoseGraph {
   virtual transform::Rigid3d GetLocalToGlobalTransform(int trajectory_id) = 0;
 
   // Returns the current optimized trajectories.
-  virtual std::vector<std::vector<TrajectoryNode>> GetTrajectoryNodes() = 0;
+  virtual MapById<NodeId, TrajectoryNode> GetTrajectoryNodes() = 0;
```

Comment:
Also drop the `mapping::` at `mapping::MapById<mapping::SubmapId, ...`?

Reply:
Done